index.html

```diff
@@ -7,37 +7,87 @@
    
   </head>
   <body>
+    <header>
+      <nav>
+        <ol>
+          <li id="AboutLink"><a href="#About">About</a></li> 
+          <li id="ExperienceLink"><a href="#Experience">Experience</a></li>
+          <li id="SkillsLink"><a href="#skills"> Skills</a></li>
+          <li id="ProjectsLink"><a href="#projects"> Projects</a></li>
+          <li id="ConnectLink"><a href="#connect"> connect</a></li>
+        </ol>
+      </nav>
+    </header>
 
     <section id="my name" style="text-align: center;">
-<h1>Farida Hyderi</h1>
-</section>
+      <h1>Farida Hyderi</h1>
+    </section>
 
     <section id="About">
-<h2>About</h2>
-<p id="introduction" >My name is Farida I am very entusiastic to learn new programming languages
-also very much interested in testing </p>
+      <h2>About</h2>
+       <p id="introduction" >My name is Farida I am very entusiastic to learn new programming languages
+        also very much interested in testing </p>
     </section>
 
     <section id = "Experience" >
-<h3>Experience</h3>
-
-      <ul>
-<li id="certification1">Cognizant Agile Methodology from Forage </li>
-<li id="certification2">Accenture Project management Forage </li>
-<li id="certification3">Mastercard - Cybersecurity Job Simulation from Forage</li>
-<li id="Project">Worked on testing projects</li>
-      </ul>
+     <h3>Experience</h3>
+
+      <!-- <ul>
+    <li id="certification1">Cognizant Agile Methodology from Forage </li>
+    <li id="certification2">Accenture Project management Forage </li>
+    <li id="certification3">Mastercard - Cybersecurity Job Simulation from Forage</li>
+    <li id="Project">Worked on testing projects</li>
+      </ul> -->
+
+       <div class="columns">
+            <!-- column 1 -->
+            <div class="column">
+                <h4>Agile Methodology </h4>
+                <p> Did small Projects featuring Agile methodologies and the tools that are used to manage projects</p>
+
+                <h4> Project management </h4>
+                <p>Small projects on how the projects are managed, hoe the meetings are held </p>
+
+                <h4>Cybersecurity Job Simulation </h4>
+                <p> Projects related to maintain security, identifying phishing emails   </p>
+
+              </div>  
+            
+            <!-- column 2 -->
+            <div class="column">
+                <h4> Dates  </h4>
+                <p>2/1/2024 to 2/28/2024</p>
+                <h4> Dates   </h4>
+                <p>3/1/2024 to 3/31/2024 </p>
+            
+               <h4>  Dates </h4>
+               <p> 4/1/2024 to 4/30/2024 </p>
+            </div> 
+            
+            
+            
+        </div> <!-- columns --> 
+    
+
     </section>
   
 
     <section id = "connect" >
-<h4>Connect</h4>
+    <h2>Connect</h2>
+
+    <ul class="links">
+    <li ><a id="linkedin" href="https://www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a></li>
```

Comment:
I don't see that flex box was used for this section or the one above it. Would you mind reworking this page using Flexbox in accordance with the assignment deliverables? Here is a pretty good reference: [A guide to flexbox](https://css-tricks.com/snippets/css/a-guide-to-Flexbox/).